robolectric/src/test/java/org/robolectric/shadows/ShadowAccessibilityManagerTest.java

```diff
@@ -266,4 +272,19 @@ public void getAccessibilityServiceList_doesNotNPE() {
                 AccessibilityServiceInfo.FEEDBACK_SPOKEN))
         .isEmpty();
   }
+
+  @Test
+  public void accessibilityManager_activityContextEnabled() {
```

Comment:
This test is pretty much the same as:

[audioManager_activityContextEnabled_applicationInstanceIsNotSameAsActivityInstance](https://github.com/robolectric/robolectric/blob/master/robolectric/src/test/java/org/robolectric/shadows/ShadowAudioManagerTest.java#L1545).

Can you refactor these tests and put them in `integration_tests/ctesque/src/androidTest/java/android/app/ContextTest.java`.

E.g. it could go into a loop, using this rough pseudo-code:

```java
ImmutableList<String> servicesToCheck = ImmutableList.of(Context.ACCESSIBILITY_SERVICE, 
(Context.AUDIO_SERVICE);

for (String service: services) {
 Object activityService = <get Activity service>;
 Object applicationService = <get APplication service>;

  assertThat(activityService).isNotSameInstanceAs(applicationService);
} 
```